src/aragonjs-wrapper.js

```diff
@@ -64,6 +64,25 @@ const appBaseUrl = (app, gateway = ipfsDefaultConf.gateway) => {
 const applyAppOverrides = apps =>
   apps.map(app => ({ ...app, ...(appOverrides[app.appId] || {}) }))
 
+const hasWebApp = app => Boolean(app['start_url'])
```

Comment:
Given that these functions are only used in `prepareFrontendApps()` it might make sense to define them there.